pkg/build/util/util.go

```diff
@@ -74,3 +75,9 @@ func NameFromImageStream(namespace string, ref *kapi.ObjectReference, tag string
 func IsBuildComplete(build *buildapi.Build) bool {
 	return build.Status.Phase != buildapi.BuildPhaseRunning && build.Status.Phase != buildapi.BuildPhasePending && build.Status.Phase != buildapi.BuildPhaseNew
 }
+
+// ConfigSelector returns a label Selector which can be used to find all
+// builds for a BuildConfig.
+func ConfigSelector(name string) labels.Selector {
+	return labels.Set{buildapi.BuildConfigLabel: name}.AsSelector()
```

Comment:
we just deprecated the "buildconfig" label in favor of an "openshit.io/build-config.name" label.  I think this code hasn't been rebased since the label was changed.  Really anywhere we're looking at the buildconfig label now, we need to look for both the old label and the new label, until we can fully drop support for the old label.

@php-coder should have been a part of your pull introducing the new label, sorry i didn't think of it at the time.  can you do a follow up?